cpp/memilio/math/stepper_wrapper.h

```diff
@@ -42,36 +58,47 @@ template <typename FP,
           class ControlledStepper>
 class ControlledStepperWrapper : public mio::OdeIntegratorCore<FP>
 {
+    using Algebra      = boost::numeric::odeint::vector_space_algebra;
+    using Operations   = typename boost::numeric::odeint::operations_dispatcher<Eigen::VectorX<FP>>::operations_type;
+    using Resizer      = boost::numeric::odeint::initially_resizer;

```

Comment:
The ControlledStepper also uses a Resizer. Do they need to match?

Reply:
The question was more towards if the template argument "Resizer" of the boost classes for the ControlledStepper and the Stepper (controlled_runge_kutta) need to be the same. They are now due to the default value of controlled_runge_kutta also being boost::numeric::odeint::initially_resizer

Reply:
Ah, okay. They don't have to be the same, but since the default for controlled_runge_kutta is to use the ErrorStepper's Resizer, they are.

We also want to use initially_resizer in both cases, since we use dynamic vectors, but do not (intend to) change their size at runtime.